internal/pkg/aws/cloudwatch/cloudwatch.go

```diff
@@ -125,6 +133,58 @@ func (cw *CloudWatch) AlarmStatuses(opts ...DescribeAlarmOpts) ([]AlarmStatus, e
 	return alarmStatuses, nil
 }
 
+// AlarmDescriptions returns the config of alarms filtered by name.
+func (cw *CloudWatch) AlarmDescriptions(alarmNames []string) ([]*AlarmDescription, error) {
+	var alarmDescriptions []*AlarmDescription
+	in := &cloudwatch.DescribeAlarmsInput{
+		AlarmNames: aws.StringSlice(alarmNames),
+	}
+	for {
+		alarmResp, err := cw.client.DescribeAlarms(in)
+		if err != nil {
+			return nil, fmt.Errorf("describe CloudWatch alarms: %w", err)
+		}
+		if alarmResp == nil {
+			break
+		}
+		alarmDescriptions = append(alarmDescriptions, cw.compositeAlarmsDescriptions(alarmResp.CompositeAlarms)...)
```

Comment:
What is the difference between composite and Metric alarm Description and which onr do we choose to display if one alarm has both the descriptions available?

Reply:
[Composite alarms](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Create_Composite_Alarm.html) are overarching alarms made up of metric or other composite alarms. Both types work for ABR, so we need to display descriptions of both types.